src/key/key.js

```diff
@@ -690,13 +690,15 @@ class Key {
         await user.verifyAllCertifications(verificationKeys, date, config) :
         [{ keyID: primaryKey.getKeyID(), valid: await user.verify(date, config).catch(() => false) }];
 
-      results.push(...signatures.map(
-        signature => ({
-          userID: user.userID.userID,
-          keyID: signature.keyID,
-          valid: signature.valid
-        }))
-      );
+      if (user.userID !== null && user.userID.userID !== null) {
```

Comment:
Rather than throwing away the result entirely for user attributes, it might be better to adapt the object below to take them into account.

Reply:
Right - thank you very much!